<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pokercode</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1" columnstretch="1,1,1,1,1,1,1,1" rowminimumheight="1,1,1,1" columnminimumwidth="1,1,1,1,1,1,1,1">
      <item row="0" column="0">
       <widget class="QToolButton" name="toolButton_s0">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>♠</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButton_s1">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toolButton_s2">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="toolButton_s3">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="toolButton_s4">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="toolButton_s5">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="toolButton_3">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>,</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QToolButton" name="toolButton_4">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="toolButton_h0">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>♥</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="toolButton_h1">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="toolButton_h2">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="toolButton_h3">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QToolButton" name="toolButton_h4">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QToolButton" name="toolButton_h5">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QToolButton" name="toolButton_5">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QToolButton" name="toolButton_6">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>!</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="toolButton_c0">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>♣</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="toolButton_c1">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="toolButton_c2">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QToolButton" name="toolButton_c3">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QToolButton" name="toolButton_c4">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QToolButton" name="toolButton_7">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QToolButton" name="toolButton_8">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QToolButton" name="toolButton_9">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QToolButton" name="toolButton_d0">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>♦</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="toolButton_d1">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="toolButton_d2">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QToolButton" name="toolButton_d3">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QToolButton" name="toolButton_d4">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QToolButton" name="toolButton_d5">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QToolButton" name="toolButton_d6">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QToolButton" name="toolButton_d7">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit_1">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="documentTitle">
         <string/>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'黑体'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>poker</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="toolButton_1">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>→</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>←</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="markdown">
         <string/>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'黑体'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>letter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
